index.html

```diff
@@ -1,120 +1,90 @@
 <!doctype html>
 <html>
-	
+
 	<head>
-		<link rel="stylesheet" href="stylesheet.css"/>
-		<link rel="shortcut icon" href="favicon.ico" type="image/x-icon"/>
+		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+		<link rel="shortcut icon" href="./images/favicon.ico" type="image/x-icon"/>
 		<title>Smelt: Batch Package Manager</title>
+		<style>
+		code {
```

Comment:
Good start - change this CSS section to:
```css
code {
    color: black;
    background-color: <something>;
    padding: <something>;
}
```
Replace `<something>` with something that looks nice :)